### ISSUE TITLE
USB to N64 Adapter Unresponsive in Games

### ISSUE THREAD

**Issue by larryebaum** [OpenEmu/OpenEmu#1779]
I'm using a [USB to N64 adapter](http://www.ebay.com/itm/131166057725?_trksid=p2059210.m2749.l2649&ssPageName=STRK%3AMEBIDX%3AIT) and connecting a N64 controller (actually N64 one, not a clone).

The OpenEmu preferences, input panel lists inputs as 'USB GamePad 1', 'USB GamePad 2' and then repeats both entries 2 more times.

If I select the first entry for 'USB GamePad 1', it appears I can map the controller -- If I select the field for 'Up' and then push the controller joystick to up, the preferences redirects to the next field. I can map other fields to any controller button.

However, once done, when I try to play any game, no controller movements/actions seem to take effect.

Relevant output for `system_profiler SPUSBDataType`

```
    USB GamePad:

      Product ID: 0x3013
      Vendor ID: 0x0e8f
      Version: 1.11
      Speed: Up to 1.5 Mb/sec
      Manufacturer: HuiJia
      Location ID: 0x14200000 / 9
      Current Available (mA): 500
      Current Required (mA): 300
```

Thoughts/Ideas?

**Comment by guitman444** [comment]
Any luck getting this to work? Having same issue.

**Comment by clobber** [comment]
Issues using adapters should be resolved as of OpenEmu 2.2.
